mmdet3d/apis/test.py

```diff
@@ -29,8 +35,42 @@ def single_gpu_test(model, data_loader, show=False, out_dir=None):
             result = model(return_loss=False, rescale=True, **data)
 
         if show:
-            model.module.show_results(data, result, out_dir)
+            # Visualize the results of MMdetection3D model
+            # 'show_results' is MMdetection3D visualization API
+            if hasattr(model.module, 'show_results'):
```

Comment:
Should use `isinstance` to determine whether the detector is a 2D detector or a 3D detector.